queue_entry.py

```diff
@@ -553,7 +552,7 @@ def execute(self):
             # Create a centred postions of the current postion
             pos_dict = self.diffractometer_hwobj.getPositions()
             cpos = queue_model_objects.CentredPosition(pos_dict)
-            pos = shape_history.Point(None, cpos, None) #, True)
+            #pos = shape_history.Point(None, cpos, None) #, True)
```

Comment:
Thanks Ivars!

To Matias,
I can only do a limit test with the mockups and thanks a lot for all the tests! Hope this time it will work…


From: Ivars Karpics [mailto:notifications@github.com]
Sent: Tuesday, February 02, 2016 10:05 AM
To: mxcube/HardwareObjects
Cc: Jie Nan
Subject: Re: [HardwareObjects] a few fixes of mockups (#9)


In queue_entry.py<https://github.com/mxcube/HardwareObjects/pull/9#discussion_r51540617>:

> @@ -553,7 +552,7 @@ def execute(self):

>              # Create a centred postions of the current postion

>              pos_dict = self.diffractometer_hwobj.getPositions()

>              cpos = queue_model_objects.CentredPosition(pos_dict)

> -            pos = shape_history.Point(None, cpos, None) #, True)

> +            #pos = shape_history.Point(None, cpos, None) #, True)

Hi,
Yes, all that block (559 - 571) can be commented. As it looks the idea was to assign new centred position to all tasks in the task group where the centring task is. In principle it is enought to center before all tasks and thats it. I dont see any reason to assign this point to tasks. but we can discuss this...

—
Reply to this email directly or view it on GitHub<https://github.com/mxcube/HardwareObjects/pull/9/files#r51540617>.